<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DementiaTest</class>
 <widget class="QDialog" name="DementiaTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="RandomQuestions">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Answer the questions</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="Next2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>33</y>
      <width>235</width>
      <height>212</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Phone number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Favorite animal:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label6">
         <property name="text">
          <string>Favorite drink:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label7">
         <property name="text">
          <string>Favorite color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label8">
         <property name="text">
          <string>favorite day</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLineEdit" name="lineEdit1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit7"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Next">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>American Typewriter</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Test for helping diagnose Alzheimer's &amp; Dementia.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="AnswerBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>222</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Input the objects that you remember:</string>
   </property>
   <layout class="QFormLayout" name="formLayout_7">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="label9">
        <property name="text">
         <string>Object 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label10">
        <property name="text">
         <string>Object 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label11">
        <property name="text">
         <string>Object 3:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <widget class="QLineEdit" name="lineEdit8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit9"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit10"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Submit">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
